judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/data/primitiveTypedElement.etl

```diff
@@ -20,6 +20,10 @@ rule CreateAttributeFromField
         log.debug("Created Attribute (Field): " + t.name);
     }
 
+
+//////////////////////////////////////////////////////////////////
+// Primitive Field Default Expression
+//////////////////////////////////////////////////////////////////   
```

Comment:
nice:D (not a change request)

Reply:
If you check the file new blocks are introduced. There is several lines added.